script/runc.spec

```diff
@@ -0,0 +1,63 @@
+##  Build runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/runc.tar.gz
+##   2. run rpmbuild -ba script/runc-centos.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _runc_sbindir /usr/local/sbin
+%global _runc_mandir  /usr/local/share/man
+%global _docdir  /usr/local/share/doc
```

Comment:
@moypray installing files into /usr/local can end up confusing users. For example:

1. user installs distro-supplied runc
2. Then user installs your (possibly newer) runc (everything goes into /usr/local) and doesn't conflict with the distro's runc. Also /usr/local probably won't be in the user's $PATH so they would still end up using the distro's runc.